src/main/scala/io/qbeast/spark/index/query/QuerySpecBuilder.scala

```diff
@@ -92,37 +93,35 @@ private[spark] class QuerySpecBuilder(sparkFilters: Seq[Expression]) extends Ser
 
     // Split conjunctive predicates
     val filters = dataFilters.flatMap(filter => splitConjunctivePredicates(filter))
-
     val indexedColumns = revision.columnTransformers.map(_.columnName)
-    val fromTo =
+
+    val (from, to) =
       indexedColumns.map { columnName =>
         // Get the filters related to the column
         val columnFilters = filters.filter(hasColumnReference(_, columnName))
 
         // Get the coordinates of the column in the filters,
         // if not found, use the overall coordinates
-        val from = columnFilters
+        val from_ = columnFilters
```

Comment:
I should change the variable names to columnsFrom and columnsTo, since the usage of _ as a prefix is a Java convention of bean getters/setters, and its usage as suffixes is discouraged:

Quoting from [Scala style guide](https://docs.scala-lang.org/style/naming-conventions.html):
> The underscore is in the prefix position rather than the suffix to avoid any danger of mistakenly typing name _ instead of name_.